docs/source/getting_started/installation.rst

```diff
@@ -53,7 +53,7 @@ You can also build and install vLLM from source:
 
     $ git clone https://github.com/vllm-project/vllm.git
     $ cd vllm
-    $ pip install -e .  # This may take 5-10 minutes.
+    $ VLLM_INSTALL_PUNICA_KERNELS=1 pip install -e .  # This may take 5-10 minutes.
```

Comment:
```suggestion
    $ # export VLLM_INSTALL_PUNICA_KERNELS=1 # optionally build for multi-LoRA capability
    $ pip install -e .  # This may take 5-10 minutes.
```

Punica installation takes a long time so generally recommend skipping if this feature is not needed.

Reply:
simon，There are two areas in the document that need to be modified。
![image](https://github.com/vllm-project/vllm/assets/5028386/239dac1c-5bed-440d-94bf-a9b6269e74b8)